packages/manager/apps/dedicated-servers/src/components/errorComponent.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ErrorBanner } from '@ovh-ux/manager-react-components';
+import { ApiError } from '@ovh-ux/manager-core-api';
+
+export type ErrorBannerProps = {
+  status?: number;
+  data?: any;
+  headers?: Record<string, string>;
+};
+
+export type ErrorProps = { error: ApiError } | { error: ErrorBannerProps };
+
+export const ErrorComponent = ({ error }: ErrorProps) => {
+  let errorObj: ErrorBannerProps;
+  if ('response' in error) {
+    const { response }: any = error;
+    errorObj = {
+      data: error,
+      headers: response?.headers,
+      status: response?.status,
+    };
+  } else {
+    errorObj = error;
+  }
+  return <ErrorBanner error={errorObj} />;
```

Comment:
```suggestion
  return <ErrorBanner error={error?.response ? {...error.response, data: error} : error} />;
```

It seems weird though : data is error.response.data, isn't it ?